cmd/ko/local.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type LocalOptions struct {
+	Local bool
+}
+
+func addLocalArg(cmd *cobra.Command, lo *LocalOptions) {
+	cmd.Flags().BoolVarP(&lo.Local, "local", "L", lo.Local,
+		"Whether to publish images to a local docker daemon vs. a registry.")
```

Comment:
Would it ever make sense to want to do both? I guess you could just drop the flag and rerun it?

Reply:
I don't think so?  You can use a registry with minikube too, but you lose the fully-local aspect.